recipes/tinyalsa/all/conanfile.py

```diff
@@ -17,17 +19,22 @@ class TinyAlsaConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def export_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
     def configure(self):
         if self.settings.os != "Linux":
             raise ConanInvalidConfiguration("Only Linux supported")
```

Comment:
Please move this check into `validate` method`.

Reply:
Sorry it is my mistake.
Fixed it.